### DISCUSSION TITLE
筷筷月报#14 - 12月复盘丨开始写日记丨软木板丨书丨漫展和对孤独的思考丨番剧和游戏

### DISCUSSION THREAD

**Discussion by giscus** [ztkuaikuai/blog#15] [General]
# 筷筷月报#14 - 12月复盘丨开始写日记丨软木板丨书丨漫展和对孤独的思考丨番剧和游戏

记录一些技术、生活、阅读的感悟；是我的人类使用说明书。

https://blog.kuaikuaitz.top/posts/%E7%AD%B7%E7%AD%B7%E6%9C%88%E6%8A%A514/

**Comment by YiGuaiShou** [comment]
推一下这个截图美化工具： https://shots.so/

**Reply by ztkuaikuai** [reply]
感谢推荐，码住！
